tests/zstreamtest.c

```diff
@@ -1294,8 +1597,22 @@ int main(int argc, const char** argv)
             result = basicUnitTests(0, ((double)proba) / 100, customMem);  /* use custom memory allocation functions */
     }   }
 
-    if (!result && !mtOnly) result = fuzzerTests(seed, nbTests, testNb, ((double)proba) / 100, bigTests);
-    if (!result) result = fuzzerTests_MT(seed, nbTests, testNb, ((double)proba) / 100, bigTests);
+    if (!result) {
```

Comment:
I changed the logic to only run `simple_api`.
In most cases, I know which API I need to test.